scripts/check_frontend_coverage.py

```diff
@@ -459,6 +459,7 @@
     'topic-editor-state.service.ts',
     'topic-editor-stories-list.directive.ts',
     'topic-editor-tab.directive.ts',
+    'topic-info-tab.directive.ts',
```

Comment:
You should not be adding any files to this list. See the note at the top of the list. /cc @marianazangrossi

Reply:
My bad on that, I was not passing checks due to I believe akshay's changes. Is there a way to push with skipping coverage tests?

Reply:
No, we strongly discourage pushing in that scenario. Talk to me if you need to do that.

Reply:
Reverted

Reply:
Done.